tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/JMSMessageTypesTest.java

```diff
@@ -158,11 +162,29 @@ public void testBytesMessageSendReceive() throws Throwable {
    }
 
    @Test(timeout = 60000)
-   public void testMessageSendReceive() throws Throwable {
+   public void testBytesMessageSendReceiveFromAMQPToAMQP() throws Throwable {
+      testBytesMessageSendReceive(createConnection(), createConnection());
+   }
+
+   @Test(timeout = 60000)
+   public void testBytesMessageSendReceiveFromCoreToAMQP() throws Throwable {
+      testBytesMessageSendReceive(createCoreConnection(), createConnection());
+   }
+
+   @Test(timeout = 60000)
+   public void testBytesMessageSendReceiveFromCoreToCore() throws Throwable {
```

Comment:
This (and the others like it) seems like it should be a duplicate of an existing test. Even if not, having it under the amqp test package doesn't seem that great.

Reply:
Its so we check cross compatibly to and from amqp. see other similar test cases, the core to core ones are just completeness on the combinations.

Reply:
I wasn't commenting about the amqp test variations, just that core to core test and the others added like it. If those aren't duplicates of existing tests I'd be surprised, and they are a waste of build time if they are. However, even if they aren't, this does not feel like the place for them, in a package directed at testing amqp support. If they fail they wont indicate any issue with the amqp support someone might reasonably think they are testing. It doesn't seem like a place where anyone would really expect to go looking for them if they did want to check for duplicates.